src/arith_uint256.cpp

```diff
@@ -179,6 +180,17 @@ std::string base_uint<BITS>::ToString() const
     return (GetHex());
 }
 
+template <unsigned int BITS>
+std::string base_uint<BITS>::ToBitString() const
```

Comment:
Add a unit test for this API with known test vectors

Reply:
removed from PR as it is unused. will put it in its own PR as ptschip suggested.